File: OpenSilver.Samples.TelerikUI/OpenSilver.Samples.TelerikUI/Other/Internal/WrapPanelWithProgressiveLoading.cs
using System.Windows.Controls;

namespace OpenSilver.Samples.TelerikUI
{
    public class WrapPanelWithProgressiveLoading : WrapPanel
    {
        public WrapPanelWithProgressiveLoading()
        {
            ProgressiveRenderingChunkSize = 3;
        }
    }
}

File: OpenSilver.Samples.TelerikUI/OpenSilver.Samples.TelerikUI/Other/Internal/WrapPanelWithProgressiveLoading.vb
﻿Imports System.Windows.Controls

Namespace OpenSilver.Samples.TelerikUI
    Public Class WrapPanelWithProgressiveLoading
        Inherits WrapPanel
        Public Sub New()
            ProgressiveRenderingChunkSize = 3
        End Sub
    End Class
End Namespace
